docs/_config.yml

```diff
@@ -60,13 +60,13 @@ categoryInfo:
     cn: 用户指南
   Core Services:
     en: Core Services
-    cn: TODO
+    cn: 核心功能
```

Comment:
google translate says this is "core functions" :D

Reply:
Haha, yeah, it's kind of core functions if translate directly. But `function` in Chinese means what an application used for. The direct translation of `services` in Chinese sounds strange. Leave it to @jiacheliu3!!

Reply:
I'd say Lu's translation makes more sense.